Apps/ValidationTests/Win32/App.cpp

```diff
@@ -11,7 +11,7 @@
 #include <filesystem>
 #include <algorithm>
 
-bool doExit = false;
+volatile bool doExit = false;
```

Comment:
who is setting `doExit` to true?

Reply:
This code is awkward. Maybe declare the doExit variable in TestUtils.h instead?

Reply:
Also, why does it get optimized out if `doExit` is being used? 🤔

Reply:
No write to that variable detected by the linker in that .cpp and you get an endless loop :)

Reply:
That's amazing! Defining the boolean in testutils and just testing it in app.cpp, I end up with the same result: boolean is not checked.

Reply:
I'm not that surprised since it's the same to the compiler. That said, using a while loop for this is not a great pattern. Maybe you should use some kind of blocking signal instead?

Reply:
Sure! Do you have something in mind for that? Is there something in Arcana?

Reply:
There is, but I don't think you need it. Try [std::condition_variable](https://en.cppreference.com/w/cpp/thread/condition_variable)?

Reply:
condition_variable is fine on win32 but on x11, XNextEvent is a blocking call and it's easier to use a boolean for exiting. I prefer to have the same exit mecanism for all the platforms. So, I'll stick with an atomic<bool> for now.

Reply:
I did another try at creating the window...and it worked! I should have kept the window manager error the first time I got it.